<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgExportPlan</class>
 <widget class="QDialog" name="dlgExportPlan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>357</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>254</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export Plan</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbxEquivalency">
     <property name="title">
      <string>Equivalency File</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbGeography">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblGeography">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Geography</string>
        </property>
        <property name="buddy">
         <cstring>cmbGeography</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QgsFileWidget" name="fwEquivalency">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="dialogTitle">
         <string>Equivalency File</string>
        </property>
        <property name="filter">
         <string notr="true">*.csv</string>
        </property>
        <property name="storageMode">
         <enum>QgsFileWidget::SaveFile</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="cbxExportEquivalency">
        <property name="text">
         <string>Export Equivalency File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbxShape">
     <property name="title">
      <string>Shapefile</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="cbxExportShape">
        <property name="text">
         <string>Export Shapefile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QgsFileWidget" name="fwShape">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="dialogTitle">
         <string>Shapefile</string>
        </property>
        <property name="filter">
         <string notr="true">*.shp</string>
        </property>
        <property name="storageMode">
         <enum>QgsFileWidget::SaveFile</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbxIncludeUnassigned">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Include unassigned geography</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbxDemographics">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Include demographics</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbxMetrics">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Include metrics</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgExportPlan</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgExportPlan</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxExportShape</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbxDemographics</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxExportShape</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbxMetrics</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxExportShape</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbxIncludeUnassigned</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxExportEquivalency</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbGeography</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxExportEquivalency</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblGeography</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
